packages/database/src/migrations/20210511042757-MoveLESMISVisualsToNewDashboards-modifies-data.js

```diff
@@ -0,0 +1,100 @@
+'use strict';
+
+import { insertObject } from '../utilities/migration';
+import { generateId } from '../utilities';
+
+var dbm;
+var type;
+var seed;
+
+/**
+ * We receive the dbmigrate dependency from dbmigrate initially.
+ * This enables us to not have to rely on NODE_PATH.
+ */
+exports.setup = function (options, seedLink) {
+  dbm = options.dbmigrate;
+  type = dbm.dataType;
+  seed = seedLink;
+};
+
+const createArrayCatAgg = async db => {
+  return db.runSql(`
+    CREATE AGGREGATE array_cat_agg(anyarray) (
+      SFUNC=array_cat,
+      STYPE=anyarray
+    );
+  `);
+};
+const deleteArrayCatAgg = async db => {
+  return db.runSql(`
+    DROP AGGREGATE array_cat_agg(anyarray)
+  `);
+};
+
+const createDashboardsFromDashboardGroups = async db => {
+  await createArrayCatAgg(db);
+  const dashboards = await db.runSql(`
+    SELECT
+      name,
+      "organisationUnitCode" AS root_entity_code,
+      array_agg(CASE "organisationLevel"
+                WHEN 'SubDistrict'
+                  THEN 'sub_district'
+                  ELSE LOWER("organisationLevel")
+                END) AS entity_types,
+      array_cat_agg(distinct "projectCodes") AS project_codes,
+      array_agg(distinct "userGroup") AS base_permission_groups,
+      concat("organisationUnitCode", '_', REPLACE(name, ' ', '_'), '_', REPLACE((array_agg(distinct "userGroup"))[1], ' ', '_')) as code
+    FROM "dashboardGroup"
+    WHERE 'laos_schools' = ANY("projectCodes")
+    GROUP BY name, "organisationUnitCode";
+  `);
```

Comment:
These SQL commands are a bit complicated because they're performing the full data conversion (aside from generating an id)

I've added screenshots to the ticket showing the result of running this migration just to show that these complicated commands are working.